tests/std/tests/GH_003676_format_large_hh_mm_ss_values/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\usual_20_matrix.lst

```

Comment:
We need to skip EDG (see #395).
```suggestion
RUNALL_INCLUDE ..\concepts_20_matrix.lst
```

I think it might be better to add the test case to the existing test file [`tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp`](https://github.com/microsoft/STL/blob/main/tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp) instead of creating a new test file.

Reply:
Thanks for the help! I'm not sure, with the tests here I'm specifically asserting that `%T` and `hh_mm_ss` produce the same formatting output, which might have value of its own?